.github/workflows/basic-ci-amd64.yaml

```diff
@@ -56,6 +56,49 @@ jobs:
         timeout-minutes: 10
         run: bash tests/integration/cri-containerd/gha-run.sh run
 
+  run-containerd-sandboxapi:
+    strategy:
+      # We can set this to true whenever we're 100% sure that
+      # the all the tests are not flaky, otherwise we'll fail
+      # all the tests due to a single flaky instance.
+      fail-fast: false
+      matrix:
+        containerd_version: ['latest']
+        vmm: ['dragonball', 'cloud-hypervisor', 'qemu-runtime-rs']
+    runs-on: ubuntu-22.04
+    env:
+      CONTAINERD_VERSION: ${{ matrix.containerd_version }}
+      GOPATH: ${{ github.workspace }}
+      KATA_HYPERVISOR: ${{ matrix.vmm }}
+      SANDBOXER: "shim"
```

Comment:
Can we add this as an entry to the matrix of `run-cri-containerd` instead of duplicating job?

Reply:
Hi @sprt 

Since the sandbox api CI only supported runtime-rs and its corresponding hypervisors, which wasn't match with the cri containerd test cases, so how do I add a sandboxer entry to  it?

Reply:
You can use the `include` directive: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/running-variations-of-jobs-in-a-workflow#expanding-or-adding-matrix-configurations